CONFIG.md

```diff
@@ -275,6 +275,55 @@ directly to
 [grunt-contrib-compass](https://github.com/gruntjs/grunt-contrib-compass)
 for this theme.
 
+### Proxying Grunt Tasks
+
+Many themes come with their own Grunt integration. In those cases you can
+configure Grunt-Drupal-Tasks for awareness of that and remove the theme
+from most handling.
+
+In order to exclude your theme from handling, specify the `proxy` property in
```

Comment:
That makes sense to me. Strike the passthru, I can't think of a compelling use case.

The one downside I see is your config example is that proxy.compile-theme is a bit of a mouthful argument, especially given it's the most common mapping themes will use. Hence my use of "default".